### PR TITLE
start v0.9.0-rc0 release, shipyard

### PR DESCRIPTION
**Pull request by mangelajo**

### CODE REVIEW

File: scripts/images.sh [hunk 1]
```diff
@@ -17,7 +17,8 @@ function pull_images() {
     for project in ${PROJECTS[*]}; do
         clone_repo
         checkout_project_branch
-        local project_version=$(_git describe --tags --exclude="${CUTTING_EDGE}" --exclude="subctl-devel" --exclude="latest")
+        local base_branch="${release['branch']:-devel}}"
+        local project_version=$(base_branch)-$(shell git rev-parse --short HEAD)
```
Comment (mkolesnik) [inline scripts/images.sh]:
```suggestion
        local project_version=$(base_branch)-$(_git rev-parse --short HEAD)
```

This should work, however I think it'd be simpler to have a simple `make print-version` command in Shipyard which'll just print this info, I'll send a patch

**Review by mkolesnik — DISMISSED** [review]
I'd rather the change to images.sh be in a separate PR, which will make it easier to see later on in the git history

File: scripts/images.sh (hunk 1 above)
Comment (mangelajo) [reply scripts/images.sh]: We have to make a release today :-)
we can continue with that. let me fix this.

Comment (mangelajo) [reply scripts/images.sh]: I agree that the print-version will be better long term :+1: but hopefully this let's us start the release

Comment (nyechiel) [reply scripts/images.sh]: https://github.com/submariner-io/shipyard/pull/524/files

**Review by nyechiel — APPROVED** [review]

**Comment by nyechiel** [comment]
Let's go!

**Review by dfarrell07 — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Pull request merged**